### PR TITLE
New Icon: Toll

### LINKED ISSUE #8198: New Icon: Toll Group
### Brand Name

Toll Group

### Website

https://www.tollgroup.com/

### Popularity Metric

The SimilarWeb.com rank is 258,785. See: https://www.similarweb.com/website/tollgroup.com/

### Official Resources for Icon and Color

Icon: https://commons.wikimedia.org/wiki/File:Toll_Group_logo.svg
Colour: `#007f7b`

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by chrimaho**
toll

**Issue:** closes #8198

**Similarweb rank:** 258,785
https://www.similarweb.com/website/tollgroup.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon as as per Company website.

### CODE REVIEW

File: icons/toll.svg
Comment (adamrusted) [inline icons/toll.svg:1]: From the SVG in the website header, there seems to be [quite a few differences](https://github.com/simple-icons/simple-icons/assets/1830867/dfc399bd-50c7-468d-a7b2-43c1847114f9) between your PR in black, and the source icon in red. Would you mind taking another look?

**Review by adamrusted — CHANGES_REQUESTED** [review]
Couple changes needed before we can merge this one in, thanks for your work to date @chrimaho - and sorry a review has taken this long!

**Comment by chrimaho** [comment]
Hi @adamrusted , thanks for the feedback 👍  
I've now fixed it up.

**Comment by chrimaho** [comment]
image

**Review by adamrusted — APPROVED** [review]

**Pull request merged**